docs/setup/settings.asciidoc

```diff
@@ -33,14 +33,20 @@ Specify the position of the subdomain the URL with the token `{s}`.
 `elasticsearch.username:` and `elasticsearch.password:`:: If your Elasticsearch is protected with basic authentication,
 these settings provide the username and password that the Kibana server uses to perform maintenance on the Kibana index at
 startup. Your Kibana users still need to authenticate with Elasticsearch, which is proxied through the Kibana server.
-`server.ssl.cert:` and `server.ssl.key:`:: Paths to the PEM-format SSL certificate and SSL key files, respectively. These
-files enable SSL for outgoing requests from the Kibana server to the browser.
+`server.ssl.enabled`:: *Default: "false"* Enables SSL for outgoing requests from the Kibana server to the browser. When set to `true`, `server.ssl.certificate` and `server.ssl.key` are required
+`server.ssl.certificate:` and `server.ssl.key:`:: Paths to the PEM-format SSL certificate and SSL key files, respectively.
+`server.ssl.keyPassphrase`:: The passphrase that will be used to decrypt the private key. This value is optional as the key may not be encrypted.
+`server.ssl.certificateAuthorities`:: List of paths to PEM encoded certificate files that should be trusted.
+`server.ssl.clientAuthentication`:: *Default: none* Controls Kibana's server behavior in regard to requesting a certificate from client connections. Valid values are `required`,
```

Comment:
Good question! It's my understanding that we only try to put the common settings in the kibana.yml, I feel like `keyPassphrase` is a contender to add in there, but I dunno how I feel about `clientAuthentication` and `certiricateAuthorities` because I don't see a majority of our users using these. What are your thoughts?